pkg/cmd/checkadmission/admission_tester.go

```diff
@@ -103,9 +117,19 @@ func (tc *admissionComparatorTest) Test(t *testing.T) {
 			results.Errors = append(results.Errors, cause.Message)
 			comparatorNameToResults[name] = results
 		}
+	} else {
+		switch {
+		case actual.Response.Result == nil:
+			t.Logf("got no response.result")
+		case actual.Response.Result.Details == nil:
+			t.Logf("got no response.result.details, but result: %v", actual.Response.Result)
+		}
+
 	}
 
+	t.Logf("got %d warnings", len(actual.Response.Warnings))
 	for _, warning := range actual.Response.Warnings {
+		t.Log(warning)
```

Comment:
Is all this logging intended to be merged? Or just for debug?

Reply:
> Is all this logging intended to be merged? Or just for debug?

I'd merge it for future generations.  it's in the test code and handy when something goes wrong.